addons/project/models/project_update.py

```diff
@@ -113,15 +110,61 @@ def _build_description(self, project):
     @api.model
     def _get_template_values(self, project):
         milestones = self._get_milestone_values(project)
+        profitability_values = self._get_profitability_values(project)
+        show_profitability = (profitability_values
+            and profitability_values.get('analytic_account_id')
+            and (profitability_values.get('costs')
+            or profitability_values.get('revenues'))
+        )
+
         return {
             'user': self.env.user,
             'project': project,
+            'show_profitability': show_profitability,
+            'profitability': profitability_values,
```

Comment:
```suggestion
            'profitability': profitability_values,
            'show_profitability': show_profitability,
```
follow the guidelines